src/NUnitFramework/tests/Constraints/CollectionContainsConstraintTests.cs

```diff
@@ -51,6 +51,32 @@ public void CanTestContentsOfObjectList()
             Assert.That(list, new CollectionContainsConstraint(item));
         }
 
+        [Test]
+        public void CanTestContentsOfDictionary()
```

Comment:
Do i keep the CollectionContainsConstraint test in CollectionContainsConstraintTests and move the Contain asserts to TestCaseSource or move everything to TestCaseSource?